pkg/client/client.go

```diff
@@ -299,11 +288,31 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, allAddresses,
 				if i+1 != len(sr.KubeProxyReplacement.Devices) {
 					devices += ", "
 				}
-
 			}
 
-			fmt.Fprintf(w, "KubeProxyReplacement:\t%s\t[%s]\t[%s]\n",
-				sr.KubeProxyReplacement.Mode, devices, strings.Join(features, ", "))
+			fmt.Fprintf(w, "Protocols: \t%s\n", strings.Join(sr.KubeProxyReplacement.Protocols, ","))
```

Comment:
These protocols are specific to hostReachableServices aren't they?

Reply:
Daniel and I talked about this -- The goal is to make the output user-friendly. `hostReachableServices` might not convey the information entirely?  Suggestions are welcome.

Reply:
Looking at the underlying code to how this is populated, the answer to my prior question is yes; they're populated to "TCP" / "TCP (limited)" based upon whether fields like `option.Config.EnableHostServicesTCP` are configured.

I agree on making it user-friendly. What confused me in this instance is that as far as I understand, the "protocols" here are specific to host services, but the CLI formatting in the PR as-is moves this option away from host services, which would imply that the status report is more broad. 

We document to the user how to specify this option (`global.hostServices.protocols` under https://docs.cilium.io/en/v1.8/gettingstarted/kubeproxy-free/#kube-proxy-hybrid-modes). If we don't very closely associate the options (same line or next lines with indentation for example) then it makes the report more ambiguous - is it still talking about the hostservices.protocols option? Or does "protocols" here mean something else?

As far as I can tell, this is still very much the exact same host services option so I would suggest that the words "host services" appears very nearby to this setting (eg same line) unless we have some intention for the meaning of "protocols" here to be more broad.